tests/foreman/api/test_errata.py

```diff
@@ -513,8 +476,62 @@ def test_positive_filter_by_envs(module_org):
     assert len(errata_library) > len(errata_env)
 
 
+@pytest.fixture(scope='module')
+def setup_content_rhel6():
+    org = entities.Organization().create()
```

Comment:
You should be able to use many of the global fixtures here (like here module_org | func_org), you can make a plan to move this to global fixtures / use global fixtures. 

You can raise a followup PR for improvements !

Reply:
Will do in the followup PR